src/app/pages/send/send-crypto-asset-form/family/stacks/hooks/use-stacks-broadcast-transaction.tsx

```diff
@@ -27,10 +32,10 @@ import { useSignTransactionSoftwareWallet } from '@app/store/transactions/transa
 
 import { convertToMoneyTypeWithDefaultOfZero } from '../../../components/confirmation/send-form-confirmation.utils';
 
-export function useStacksBroadcastTransaction(unsignedTx: string) {
+export function useStacksBroadcastTransaction(unsignedTx: string, token: CryptoCurrencies = 'STX') {
   const { setActiveTabActivity } = useHomeTabs();
   const signSoftwareWalletTx = useSignTransactionSoftwareWallet();
-  const stxMarketData = useCryptoCurrencyMarketData('STX');
+  const tokenMarketData = useCryptoCurrencyMarketData(token);
```

Comment:
`useCryptoCurrencyMarketData` only works with some currencies, and definitely not many SIP10 tokens. 

What happens when no token market data is found?

Reply:
price amount is NaN
```{
    "pair": {
        "base": "ALEX",
        "quote": "USD"
    },
    "price": {
        "amount": "NaN",
        "symbol": "USD",
        "decimals": 2
    }
}```